build/media-suite_compile.sh

```diff
@@ -582,6 +582,23 @@ if [[ $ffmpeg != "no" ]] && enabled libsoxr; then
     fi
 fi
 
+if [[ $ffmpeg != "no" ]] && enabled libcodec2 && do_pkgConfig "codec2 = 0.7"; then 
+    _check=(libcodec2.a codec2.pc) 
+	if do_wget -h 0695bb93cd985dd39f02f0db35ebc28a98b9b88747318f90774aba5f374eadb2 \
+            "https://freedv.com/wp-content/uploads/sites/8/2017/10/codec2-0.7.tar.xz"; then
+        do_uninstall include/codec2 "${_check[@]}"
+		touch cmake/GetPrerequisites.cmake
```

Comment:
Normalize these tabs/spaces. 4 space indent only for each indentation level.

Reply:
I'm just using tabs in Notepad++ as always but anyways ....